common/culture/traditions/00_regional_traditions.txt

```diff
@@ -923,109 +942,91 @@ tradition_desert_nomads = {
 	}
 }
 
-# tradition_mountaineer_ruralism = {
-	# category = regional
+tradition_mountaineer_ruralism = {
+	category = regional
 
-	# layers = {
-		# 0 = martial
-		# 1 = western
-		# 4 = mountain.dds
-	# }
+	layers = {
+		0 = martial
+		1 = western
+		4 = mountain.dds
+	}
 	
-	# is_shown = {
-		# like_heritage_west_slavic_trigger = yes
-	# }
+	is_shown = {
+		# Warcraft
+		OR = {
+			has_cultural_pillar = heritage_northern
+			any_culture_county = {
+				title_province = { 
+					geographical_region = world_northrend 
+				}
+			}
+		}
+	}
 	
-	# can_pick = {
-		# like_heritage_west_slavic_trigger = yes
-		# custom_description = {
-			# text = culture_in_mountains_desc
-			# any_culture_county = {
-				# any_county_province = {
-					# OR = {
-						# terrain = mountains
-						# terrain = desert_mountains
-					# }
-				# }
-			# }
-		# }
-	# }
+	can_pick = {
+		custom_description = {
+			text = culture_in_hills_desc
+			any_culture_county = {
+				percent >= 0.5
+				any_county_province = { terrain = hills }
+			}
+		}
+	}
```

Comment:
The tradition is changed a lot so it's better to create a new one.

Reply:
I see your point, although since this modded vanilla tradition is unique to a specific culture/region (Grizzly Hills) and the base vanilla tradition is not used by any other culture (Czech mountaineers), there should be no issues with its current form.

Are there tweaks to the function or localisation of the tradition you want to see? If you want to restore `unlock_maa_zbrojnosh` (currently removed) as a wc trapper MaA, I can make a separate issue for that.

Reply:
> I see your point, although since this modded vanilla tradition is unique to a specific culture/region (Grizzly Hills) and the base vanilla tradition is not used by any other culture (Czech mountaineers), there should be no issues with its current form.

Got it, yeah, we may keep it.

> Are there tweaks to the function or localisation of the tradition you want to see? If you want to restore unlock_maa_zbrojnosh (currently removed) as a wc trapper MaA, I can make a separate issue for that.

Originally, I made it only for northern humans. Thus the ones that live in hills and taige, that's correct, but it's strange for me that it's a tradition unlocked by region.

You may keep `unlock_maa_zbrojnosh`. I may fit for them. Don't mind it.

And there's a bug. Description mentions only stoic while it's communal too.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/207108130-3360a4ec-c68e-4af8-b1d6-70085d825fcf.png)

</details>

Reply:
> Originally, I made it only for northern humans. Thus the ones that live in hills and taige, that's correct, but it's strange for me that it's a tradition unlocked by region.

I narrowed the conditions in bcf02354e to the Grizzly Hills region and character realm.